helmtestserver/server.go

```diff
@@ -69,9 +75,56 @@ func (s *HelmServer) PackageChart(path string) error {
 // with the given version, to be served by the HelmServer. It returns an
 // error in case of a packaging failure.
 func (s *HelmServer) PackageChartWithVersion(path, version string) error {
+	return s.packageChart(path, version, "")
+}
+
+// PackageSignedChartWithVersion attempts to package the chart at the given path
+// with the given version and sign it using a PGP keyring, to be served by the HelmServer.
```

Comment:
Maybe good to mention that it's a generated keyring.

Reply:
Would be also nice to talk about the `publicKeyPath`. That it write the public key of the signed package, which can be used to verify the package.